docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,255 @@
+= Deploy Blue-Green Clusters
+:description: This tutorial guides you on how you can use Hazelcast's blue-green deployment feature for your client to operate continuously when its connected cluster is down.
+
+{description} The feature provides a client failover mechanism for rerouting the client connections to a different cluster without requiring a
+client network configuration update or client restart. You can use it to manually redirect client traffic in order to perform maintenance or software updates,
+as well as automatically redirect client traffic to a different cluster during a disaster recovery scenario.
+
+This tutorial focuses on the disaster recovery scenario, and you will:
+
+. Start an Enterprise cluster (blue)
+. Start another Enterprise cluster (green)
+. Configure a Java client to have the Blue-Green deployment's failover feature
+. Connect this client to the first cluster
+. Kill the first cluster and see that the client connects to the second one
+
+NOTE: Blue-Green deployment is currently supported by Hazelcast's Java, Node.js and Go clients.
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Docker
```

Comment:
Users of these tutorials probably do not need links to Docker installation on each page. I think we can remove these.